src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -26,8 +26,18 @@
                 class="form-control"
                 rows="2"
             ></textarea>
+            <div v-if="isFeatureLine">
+                <font-awesome-icon :icon="['far', 'square']" />
+                {{ lengthRounded }} {{ length > 100 ? 'km' : 'm' }}
+            </div>
+            <div v-if="isFeaturePolygon">
+                <font-awesome-icon
+                    :icon="['far', 'square']"
+                    style="background: #888a85; color: #888a85"
+                />
```

Comment:
I would prefer that you use some kind of Bootstrap helper class here, `text-dark` or `bg-dark`, instead of hardcoding CSS values in the HTML.
If Bootstrap doesn't have the color you want, I would then prefer that you add a CSS class to this element, and style it under in the `<style>` portion of the Component